CMakeLists.txt

```diff
@@ -297,11 +301,14 @@ if(NOT WIN32)
 set_target_properties(stellarsolverstatic PROPERTIES OUTPUT_NAME stellarsolver)
 endif(NOT WIN32)
 
-install(TARGETS stellarsolverstatic DESTINATION "${CMAKE_INSTALL_PREFIX}/lib/")
-install(TARGETS stellarsolver DESTINATION "${CMAKE_INSTALL_PREFIX}/lib/")
+# What is this for??
```

Comment:
I have found that sometimes CMAKE_INSTALL_LIBDIR doesn't work properly on some systems.  This is a backup method to get it to the right place so that it can get installed and then found

Reply:
At least on Linux it shouldn't be used since LIBDIR is properly defined. Maybe wrap it up on the system where this is an issue?